examples/react/css/src/toggle-refinement.css

```diff
@@ -0,0 +1,44 @@
+.ais-ToggleRefinement-label {
+  font-size: 0.875rem;
+  line-height: 1.75rem;
+  display: block;
+  cursor: pointer;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+input[class^='ais-'][class*='ToggleRefinement-checkbox'] {
```

Comment:
I'm not sure why we need such selector, why not target `.ais-ToggleRefinement-checkbox`?

Reply:
This is because there is a selector in `common.css` that is of higher specificity `input[class^='ais-'][type='checkbox']:checked { /* ... */ }`

I did it this way because we were out of time so just needed something quick that would work but the class above probably needs to be merged into the `refinement-list.css`

Reply:
@sarahdayan I have re-worked so it uses the default class name `.ais-ToggleRefinement-checkbox`